### PR TITLE
feat: count_min_max and skip_count_min_max

### PR DESCRIPTION
**Pull request by ordian**
This PR doesn't deprecate count and skip_count.

### CODE REVIEW

File: src/combinator.rs [hunk 1]
```diff
@@ -624,6 +624,107 @@ parser!{
...
+    type Output = F;
+
+    #[inline]
+    fn parse_lazy(&mut self, input: P::Input) -> ConsumedResult<F, P::Input> {
+        let mut iter = self.parser.by_ref().iter(input.clone());
+        let mut len = 0usize;
+        let value = iter.by_ref().take(self.max).inspect(|_| { len += 1 }).collect();
+        if len < self.min {
+            let message = Error::Message(format!("expected {} more elements", self.min - len).into());
+            ConsumedErr(ParseError::new(input.position(), message))
```
Comment (ordian) [inline src/combinator.rs]: I guess position here is wrong.

Comment (ordian) [inline src/combinator.rs]: Maybe we should also track position in inspect?

Comment (Marwes) [inline src/combinator.rs]: yep, `iter.input.position()` should be correct

File: src/combinator.rs [hunk 2]
```diff
@@ -624,6 +624,107 @@ parser!{
...
+    CountMinMax {
+        parser: parser,
+        min: min,
+        max: max,
+        _marker: PhantomData,
+    }
+}
+
+pub type SkipCountMinMax<P> = skip_count_min_max::SkipCountMinMax<P>;
+parser!{
```
Comment (Marwes) [inline src/combinator.rs]: 👍

File: src/combinator.rs (hunk 1 above)
Comment (Marwes) [inline src/combinator.rs]: No need to track it in inspect as the position in iter.input should be all that we need I think

**Comment by Marwes** [comment]
Looks good to me! `count` could be implemented in terms of `count_min_max` but I will refactor that in 3.0 to reduce the amount of churn in the code.

**Comment by Marwes** [comment]
gah I need to make travis.sh executable (on windows locally which do not have that distinction)

**Pull request merged**